solr/modules/ltr/src/java/org/apache/solr/ltr/feature/FieldValueFeature.java

```diff
@@ -177,7 +177,8 @@ public FieldValueFeatureScorer(
       public float score() throws IOException {
 
         try {
-          final Document document = context.reader().document(itr.docID(), fieldAsSet);
+          final Document document =
+              context.reader().storedFields().document(itr.docID(), fieldAsSet);
```

Comment:
I think this would be good. Scorer should already not be thread-safe, and should not be called from multiple threads; and because of the way the `score()` method is called, this would I think be effectively inside a loop. FWIW I can't think of any problem calling `context.reader()` in the ctor.